Damage.cs

```diff
@@ -17,49 +16,54 @@ public EDamageTypes DamageType
                 return damageType;
             }
         }
-        #endregion
 
-        #region TargetAttribute
         public struct Target
         {
             public float Armor;
 
             public ERA ERAData;
+
+            public float Health;
         }
 
         public struct ERA
         {
-            public float currentValue;
+            public float CurrentValue;
 
             public float KEFractionMultiplier;
 
-            public float HEATFractionMultiplier;
+            public float HeatFractionMultiplier;
         }
-        #endregion
 
-        public Damage(EDamageTypes dt)
+        protected Target _target;
+
+        public Damage(EDamageTypes dt, Target t)
```

Comment:
Parameters should have more concise and meaningful names.

Rename `dt` and `t` to `damageType` and `target`.